[2025-10-08T11:27:03.039+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-08T11:27:03.075+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag1.test_pwd scheduled__2025-10-08T11:26:00+00:00 [queued]>
[2025-10-08T11:27:03.087+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag1.test_pwd scheduled__2025-10-08T11:26:00+00:00 [queued]>
[2025-10-08T11:27:03.087+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-10-08T11:27:03.104+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): test_pwd> on 2025-10-08 11:26:00+00:00
[2025-10-08T11:27:03.113+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=129) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-08T11:27:03.114+0000] {standard_task_runner.py:63} INFO - Started process 132 to run task
[2025-10-08T11:27:03.115+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag1', 'test_pwd', 'scheduled__2025-10-08T11:26:00+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp8clclebr']
[2025-10-08T11:27:03.118+0000] {standard_task_runner.py:91} INFO - Job 23: Subtask test_pwd
[2025-10-08T11:27:03.134+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-10-08T11:27:03.181+0000] {task_command.py:426} INFO - Running <TaskInstance: dag1.test_pwd scheduled__2025-10-08T11:26:00+00:00 [running]> on host 567e6f4df55c
[2025-10-08T11:27:03.340+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag1' AIRFLOW_CTX_TASK_ID='test_pwd' AIRFLOW_CTX_EXECUTION_DATE='2025-10-08T11:26:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-08T11:26:00+00:00'
[2025-10-08T11:27:03.343+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-08T11:27:03.344+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-10-08T11:27:03.345+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'pwd']
[2025-10-08T11:27:03.369+0000] {subprocess.py:86} INFO - Output:
[2025-10-08T11:27:03.373+0000] {subprocess.py:93} INFO - /tmp/***tmpvxyo7r49
[2025-10-08T11:27:03.374+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-10-08T11:27:03.375+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-08T11:27:03.413+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=test_pwd, run_id=scheduled__2025-10-08T11:26:00+00:00, execution_date=20251008T112600, start_date=20251008T112703, end_date=20251008T112703
[2025-10-08T11:27:03.489+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-08T11:27:03.515+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-08T11:27:03.517+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
